docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_mi_ad_auth_arm_template/_index.md

```diff
@@ -0,0 +1,224 @@
+  - _`deployPostgreSQL`_ - Boolean that sets whether or not to deploy PostgreSQL, for this scenario. Default value is _**false**_.
+  - _`deployBastion`_ - Boolean that sets whether to deploy Azure Bastion or not to connect to the client VM. Default value is _**false**_.
+  - _`bastionHostName`_ - Azure Bastion host name. Default value is _**Arc-Data-Demo-Bastion**_.
+
+    > **NOTE: In case you decided to deploy SQL Managed Instance in an highly-available fashion, refer to the ["High Availability"](##-High-Availability-with-Always-On-availability-groups) section in this readme. Also note that this capability is currently in [preview](https://docs.microsoft.com/azure/azure-arc/data/release-notes)**.
+
+- To deploy the ARM template, navigate to the local cloned [deployment folder](https://github.com/microsoft/azure_arc/blob/main/azure_arc_data_jumpstart/aks/ARM) and run the below command:
+
+    ```shell
+    az group create --name <Name of the Azure resource group> --location <Azure Region>
+    az deployment group create \
+    --resource-group <Name of the Azure resource group> \
+    --name <The name of this deployment> \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_data_jumpstart/aks/ARM/azuredeploy.json \
+    --parameters <The *azuredeploy.parameters.json* parameters file location>
+    ```
+
+    > **NOTE: Make sure that you are using the same Azure resource group name as the one you've just used in the _`azuredeploy.parameters.json`_ file**
+
+    For example:
+
+    ```shell
+    az group create --name Arc-Data-Demo --location "East US"
+    az deployment group create \
+    --resource-group Arc-Data-Demo \
+    --name arcdata \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_data_jumpstart/aks/ARM/azuredeploy.json \
+    --parameters azuredeploy.parameters.json
+    ```
+
+    > **NOTE: The deployment time for this scenario can take ~15-20min**
+
+- Once Azure resources have been provisioned, you will be able to see them in the Azure portal. At this point, the resource group should have **13 various Azure resources** deployed (If you chose to deploy Azure Bastion, you will have **14 Azure resources**).
+
+    ![Screenshot showing ARM template deployment completed](./01.png)
+
+    ![Screenshot showing the new Azure resource group with all resources](./02.png)
+
+## Windows Login & Post Deployment
+
+- Now that the first phase of the automation is completed, it is time to RDP to the client VM. If you have not chosen to deploy Azure Bastion in the ARM template, RDP to the VM using its public IP.
+
+    ![Screenshot showing the Client VM public IP](./03.png)
+
+- If you have chosen to deploy Azure Bastion in the ARM template, use it to connect to the VM.
```

Comment:
Point out that for Bastion to login in using a domain account , the format needs to be <username>@domain.com

Reply:
Fixed